src/store/reducers/todo.js

```diff
@@ -1,15 +1,29 @@
+import dayjs from "dayjs";
 import { ADD_TASK } from "../constants/actionTypes";
 
 const initialTodoState = {
-  tasks: [],
+  tasks: [
+    {
+      id: `task#${dayjs().format("YYYY-MM-DD HH:mm:ss")}`,
```

Comment:
We can use [uuid](https://www.npmjs.com/package/uuid) V4 for generating id instead of dates. 🚀